src/src/utils/getConfig.js

```diff
@@ -0,0 +1,41 @@
+import axios from 'axios';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Supershift/Sushi.Mediakiwi.UI/pullRequest?prid=8211123)
```suggestion
import axios from "axios";
```